gcampus/analysis/static_src/js/calibration.js

```diff
@@ -139,11 +148,11 @@ function createChart(pk, el, formula, title, x_label, y_label) {
 function updateAnnotation(x, y, chart) {
     // Needs calculation since the chart.js x values do not correspond
     // to the xdata but has to be scaled relative to the axis labels
-    let len = chart.data.labels.length -1;
+    let len = chart.data.labels.length - 1;
     let x_min = parseFloat(chart.data.labels[0]);
     let diff = parseFloat(chart.data.labels[len]) - x_min;
-    console.log(x, y, x_min, len, ((x-x_min)/diff)*len);
-    chart.options.plugins.annotation.annotations.point1.xValue = ((x-x_min)/diff)*len;
+    console.log(x, y, x_min, len, ((x - x_min) / diff) * len);
```

Comment:
Can you remove the `console.log` here?

Reply:
fixed in f4ff7ac108071bff03927496da4eca6751639a20